tasks/update-infra-deployments/update-infra-deployments.yaml

```diff
@@ -65,25 +65,26 @@ spec:
           mountPath: /shared
       workingDir: /shared
       script: |
+        #!/usr/bin/env bash
         set -x
-        cd cloned
+        cd cloned || exit
```

Comment:
Did shellcheck make you do this?

BTW, if shebang is not specified, tekton will use:
```
#!/bin/sh
set -e
```
So you should do `set -ex`, or even better `set -eux`  (but that is a change in behavior and it could potentially break something).

Maybe with that you won't need the `|| exit`?

Reply:
Yes, shellcheck made me. I will try those things out

Reply:
I added `-e` to the set and removed the `|| exit` and it passes. Sweet